### PR TITLE
SOSSE-9021: Extended Events minimum duration enhancement

### PR DESCRIPTION
**Pull request by CharlesWang1**
https://jira.labs.quest.com/browse/SOSSE-9025

### CODE REVIEW

File: pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md [hunk 1]
```diff
@@ -17,4 +17,5 @@ DS will send a resolve event to PagerDuty when alarm is cleared automatically. |
 The Wait Events and Workload Analysis drilldowns have been enhanced to show the procedure name instead of the SQL text. | SEMB-289, SOSSE-8805
 The I/O stall alarm rule has been enhanced to reduce the alarm noise. | SEMB-561, SOSSE-8981
 The Recompiles alarm rule has been enhanced to reduce number of alarms. | SEMB-560, SOSSE-8980
+Updated the default Extended Events minimum duration from 0 to 200. | SEMB-585 , SOSSE-9021
```
Comment (RGriffin-Quest) [inline pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md:20]: I think we need to explain how this change might result in less data being seen in the Wait Events and Workload Analysis drilldowns.

**Review by RGriffin-Quest — CHANGES_REQUESTED** [review]

File: pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md (hunk 1 above)
Comment (CharlesWang1) [reply pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md]: @RGriffin-Quest Please review again, Thanks

Comment (CharlesWang1) [reply pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md]: Sorry, you mean how, Will change

Comment (CharlesWang1) [reply pages/SpotlightEnterprise/enterprise_releasenotes_enhancements.md:20]: @RGriffin-Quest As talked with @CarolineHuang keep the description as before

**Pull request merged**